PhysicsTools/NanoAOD/plugins/IsFromLostTrackMapProducer.cc

```diff
@@ -0,0 +1,139 @@
+// -*- C++ -*-
+//
+// Package:    PhysicsTools/NanoAOD
+// Class:      IsFromLostTrackMapProducer
+//
+/**\class IsFromLostTrackMapProducer IsFromLostTrackMapProducer.cc PhysicsTools/NanoAOD/plugins/IsFromLostTrackMapProducer.cc
+
+ Description: [one line class summary]
+
+ Implementation:
+     [Notes on implementation]
+*/
+//
+// Original Author:  Maria Giulia Ratti (ETHZ) [mratti]
+//         Created:  Thu, 22 Nov 2018 12:34:48 GMT
+//
+//
+
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/global/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+#include "DataFormats/PatCandidates/interface/IsolatedTrack.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h" 
+
+//
+// class declaration
+//
+
+template <typename T>
```

Comment:
should not be a templated class as its implementation is specific to pat::IsolatedTrack